### DISCUSSION TITLE
Unleash proxy client for Angular

### DISCUSSION THREAD

**Discussion by yuri-karelics** [Unleash/unleash#3621] [Show and tell]
Hi, we in [Karelics](https://karelics.fi/) team have integrated feature flags into our Cloud solution. Angular proxy client we found on your web site was a bit outdated at the moment, so we implemented our own. Here it is https://www.npmjs.com/package/@karelics/angular-unleash-proxy-client
Also we prepared short blogpost/tutorial about how it was https://karelics.fi/unleash-feature-flags-with-gitlab/
Hope it would be helpful for somebody.
If you find our NPM package interesting we are ok if you mention it in the documentation "Community SDKs" section. We are planing to maintain it in future. If you have ideas what additional parts of Unleash functionality can be covered in our client please give us know and feel free to contribute.

**Comment by pransh15** [comment]
Hey @yuri-karelics 👋

Thanks for sharing this! I will tag in @ivarconr, Founder and CTO at Unleash to take a look. 👀

If all is good, you can submit a PR simply by editing the docs here: [community SDKs](https://github.com/Unleash/unleash/edit/main/website/docs/reference/sdks/index.md) 🔗

**Reply by sighphyre** [reply]
Hey @yuri-karelics! Oh this is very cool! I'll need a little bit of time to read the code and test this but at first glance this looks great and I don't see any reason why we shouldn't mention this in our docs, let me chat with the rest of the engineering team before I make commitments though. 

The good news is that Proxy SDKs are super simple, if it supports resolving a feature toggle, resolving a variant and sending metrics back to the Proxy, then you're pretty much done and we have no plans to add new features to our Proxy SDK feature set. Guessing you haven't done metrics here, since it looks like you're using Gitlab but if the rest of team likes it, I'm fairly sure we'd be open to sending a PR your way.

Thanks so much for sharing this!

**Reply by yuri-karelics** [reply]
Hi @pransh15 @sighphyre thank you for your feedback! Yes, we just implemented simple Angular wrapper around your proxy client. And yes, we are using GitLab and its Unleash integration looks a bit limited, so only GitLab related features are covered. If you can show example in other SDKs what "metrics" means we are open to add some code into our repo, and we are open for contributions :) In general, we just hope this library would be helpful for other Angular developers who is using GitLab/Unleash for feature flags. The benefit we have agains existing Angular client mentioned in your documentation - we are using our lib in our production, so we will definitely care about proper maintenance and updates.

**Reply by ivarconr** [reply]
This is great @yuri-karelics. Thanks for building this and writing a blog about it! 🚀 

The fact that its built on top of the pure Javascript SDK, should take care of the metrics part automatically. Feel free to add it in the list of community provided SDKs.
